lib/signs/utilities/last_trip.ex

```diff
@@ -10,7 +10,13 @@ defmodule Signs.Utilities.LastTrip do
     case service_status do
       {has_top_service_ended?, has_bottom_service_ended?} ->
         {unpacked_mz_top, unpacked_mz_bottom} = unpack_mezzanine_content(messages, source)
-        {top_source, bottom_source} = source
+        {top_source_config, bottom_source_config} = source
+
+        routes =
+          Enum.flat_map(
+            top_source_config.sources ++ bottom_source_config.sources,
+            &List.wrap(&1.routes)
+          )
```

Comment:
We have a helper method [here](https://github.com/mbta/realtime_signs/blob/main/lib/signs/utilities/source_config.ex#L193-L195) that should let you do something like
```suggestion
        routes = Signs.Utilities.SourceConfig.sign_routes(source)
```